CMake/SlicerExtensionCPack.cmake

```diff
@@ -59,6 +59,14 @@ endforeach()
 include(SlicerMacroExtractRepositoryInfo)
 SlicerMacroExtractRepositoryInfo(VAR_PREFIX ${EXTENSION_NAME})
 
+#-----------------------------------------------------------------------------
+# Set extension revision
+#-----------------------------------------------------------------------------
+if(NOT "${${EXTENSION_NAME}_FORCED_REVISION}" STREQUAL "")
```

Comment:
Should we add this `...FORCED...` variable to the CACHE? If we configure the build with `-DSandbox_FORCED_WC_REVISION=300` but then we configure again without this option then is the forced revision going to be preserved?